src/dotnet/APIView/APIView/NamespaceAPIV.cs

```diff
@@ -48,10 +48,17 @@ public NamespaceAPIV(INamespaceSymbol symbol)
 
         public override string ToString()
         {
-            var returnString = new StringBuilder();
             var renderer = new TextRendererAPIV();
-            renderer.Render(this, returnString);
-            return returnString.ToString();
+            var list = new List<LineAPIV>();
+            renderer.Render(this, list);
+
+            var builder = new StringBuilder();
```

Comment:
Not a fan of this code being duplicated many times.

Reply:
Add a type `StringList` that inherits from List<string> and override ToString() method on it to concat lines. Use it instead of List<string> everywhere